packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5910,8 +5914,15 @@ export default class Meeting extends StatelessWebexPlugin {
    * @public
    * @memberof Meeting
    */
-  public leave(options: {resourceId?: string; reason?: any} = {} as any) {
+  public leave(
+    options: {
+      resourceId?: string;
+      reason?: any;
```

Comment:
Looks like `reason` is now not used for anything? its value is copied into `leaveReason`, but then `leaveReason` is not passed anywhere further, only logged in 1 place: in line 5978 and we should probably log `options.clientEventLeaveReason` there instead, right? 
Or am I missing something?

Reply:
The "reason" arg is now only used for logging. Several places that call meeting.leave() pass in a specific reason, and two other places use the reason from locus events (EVENTS.DESTROY_MEETING, EVENTS.REMOTE_RESPONSE).

The "clientEventLeaveReason" is used for the CA event leaveReason. It must be one of three values. With this change, the "clientEventLeaveReason" will not be anything other than undefined (which is perfectly fine, it's optional), but I will open another Jira to see if we should make use of this arg from some of our meeting.leave() calls